core/src/main/java/tech/tablesaw/io/FileReader.java

```diff
@@ -64,7 +67,9 @@ private String cleanName(String name) {
     return name.trim();
   }
 
-  /** Returns the column names for each column in the source. */
+  /**
```

Comment:
We have an autoformatting plugin setup. If you compile on the command line just once it will automatically run on git commit. The fact that your formatting is changing makes me think the formatting plugin hasn't run on this PR

Reply:
Was this set up recently? These commits are more than a month old (I was not able to make the PR earlier). I just rebased the code recently. Let's see if it works when I split this PR into separate PRs.